src/shop/ias_gloves.py

```diff
@@ -98,7 +99,24 @@ def shop_loop(self):
                     mouse.click(button="right")
                     Logger.info("GG gloves bought!")
                     self.gloves_bought += 1
-                    time.sleep(1)
+                    wait(1, 3)
+
+                # Enable the following, if you are fine with +2 gloves.
+                else:
+                    if self.look_also_for_plus_2_gloves is True:
+                        g_gloves_found, pos = self._template_finder.search(
+                            ref=load_template(
+                                "assets/shop/gloves/g_gloves.png", 1.0 # +2 java gloves are better than no java gloves.
+                            ),
+                            inp_img=img,
+                            threshold=0.80,
+                            normalize_monitor=True,
+                        )
+                        if g_gloves_found:
+                            mouse.click(button="right")
+                            Logger.info("G gloves bought!")
+                            self.gloves_bought += 1
+                            wait(1, 3)
```

Comment:
This code is redundent with L99-L102. Rather searchfor +2 gloves in case gg_gloves not found and the config is set. Then have an or for actually buying it. e.g.:
```python
if self.look_also_for_plus_2_gloves and not gg_gloves_found:
  # search for +2 gloves
if gg_gloves_found or g_gloves_found:
  # buy gloves
```
Also I like verbose variable names, but could maybe be a little shortned. At the very least I would remove the "also" from the variable name.